http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -227,12 +227,12 @@ class LeoPubsubMessageSubscriber[F[_]](
                                 4.5 minutes
       )
       messageType = event.msg.messageType.asString
-      metricsName = e match {
+      (metricsName, tags) = e match {
         case Some(e) =>
-          s"pubsub/fail/${messageType}/${e.getClass.toString.split('.').last}"
-        case None => s"pubsub/ack/${messageType}"
+          (s"pubsub/fail", Map("messageType" -> messageType, "exception" -> e.getClass.toString.split('.').last))
+        case None => (s"pubsub/success/${messageType}", Map("messageType" -> messageType))
```

Comment:
@cpate4  I was looking to add this metrics in a chart, but think it'll be easier to do so if the exception info and the messageType are labels instead of having them in metrics name. In grafana, with the current metrics name, we'll need to add multiple queries in order to cover all message types and exceptions. If we use labels instead, we can use just less query depending on how you want to view to data